sprites.py

```diff
@@ -75,39 +79,49 @@ def __init__(self, game, species, x, y, LorR):
         self.LorR = LorR
 
         # this is the image you see most of the time
-        self.original = pygame.Surface((int(40 * self.game.scale), int(40 * self.game.scale))).convert()
+        self.original = pygame.Surface(
+            (int(40 * self.game.scale), int(40 * self.game.scale))).convert()
         self.original.fill(yellow)
-        pygame.draw.circle(self.original, black, (int(20 * self.game.scale), int(20 * self.game.scale)), int(20 * self.game.scale))
+        pygame.draw.circle(
+            self.original,
+            black,
+            (int(20 * self.game.scale), int(20 * self.game.scale)),
+            int(20 * self.game.scale))
         self.original.set_colorkey(yellow)
         self.rect = self.original.get_rect()
 
         # this is the one you see when the mouse is hovering
         # over it and it's in the escape area
         self.selected_img = self.original.copy()
-        pygame.draw.circle(self.selected_img, grey,(int(20 * self.game.scale), int(20 * self.game.scale)), int(17 * self.game.scale))
+        pygame.draw.circle(
+            self.selected_img,
+            grey,
+            (int(20 * self.game.scale), int(20 * self.game.scale)),
+            int(17 * self.game.scale))
 
         # let the original image be the one we start off with
         self.image = self.original.copy()
 
         # let's also let everyone know what kind of animal we are
-        text = Text(str(self.species + 1), size = int(20 * self.game.scale), color = white)
+        text = Text(
+            str(self.species + 1), size=int(20 * self.game.scale), color=white)
         text.rect.center = self.rect.center
         self.text = Group((text))
         self.text.draw(self.image)
-        
+
         if self.LorR == 'left':
             self.rect.center = (x - int(21 * self.game.scale), y)
         else:
             self.rect.center = (x + int(21 * self.game.scale), y)
-    
+
     def update(self):
         m = pygame.mouse.get_pos()
         d = self.game.dist(m[0], m[1], self.rect.centerx, self.rect.centery)
         button1, button2, button3 = pygame.mouse.get_pressed()
         a = self.game.escArea.prisoners
         if ((d <= int(20 * self.game.scale)) and
-            self in a.sprites() and 
-            not self.game.guard.moving):
+            self in a.sprites() and not
+           self.game.guard.moving):
```

Comment:
The old method is better as it's more readable, shows the variable the not is being applied on where it's used.

Reply:
Agree, reverted